polardrift_toolwin.cpp

```diff
@@ -250,6 +260,17 @@ void PolarDriftToolWin::OnHemi(wxCommandEvent& evt)
     FillPanel();
 }
 
+void PolarDriftToolWin::OnMirror(wxCommandEvent& evt)
+{
+    int i_mirror = m_mirrorCheck->IsChecked()? -1: 1;
+    pConfig->Profile.SetInt("/PolarDriftTool/Mirror", i_mirror);
+    if (i_mirror != m_mirror)
```

Comment:
this is fine, but could be simply
```suggestion
    m_mirror =  i_mirror;
```